traefik/values.yaml

```diff
@@ -261,7 +263,8 @@ providers:
     # -- Allows Traefik to automatically watch for file changes
     watch: true
     # -- File content (YAML format, go template supported) (see https://doc.traefik.io/traefik/providers/file/)
-    content: ""
+    content:
```

Comment:
It's not very consistent with the changes you did on tests line 291 for instance. Don't you think ?

Reply:
@darkweaver87 I reverted the changes introduced by my autoformatter, they were not intended.